eos-migrate-firefox-profile

```diff
@@ -39,6 +39,61 @@ PROFILES_INI = os.path.join(HOME_DOT_MOZILLA, "firefox", "profiles.ini")
 ENDLESS_FLATPAK_INSTALL_GROUP = "InstallFF6C2EBF42BF07E5"
 MOZILLA_FLATPAK_INSTALL_GROUP = "InstallCF146F38BCAB2D21"
 
+MIMEAPPS_LIST = os.path.join(GLib.get_user_config_dir(), "mimeapps.list")
+MIMEAPPS_GROUPS = [
+    "Default Applications",
+    "Added Associations",
+    "Removed Associations",
+]
+OLD_DESKTOP_FILE = "org.mozilla.Firefox.desktop"
+NEW_DESKTOP_FILE = "org.mozilla.firefox.desktop"
```

Comment:
This is fine for a single desktop file to migrate, but might want a dict or a list of tuples to make it easy to add to in the future.

Reply:
In principle it would have been good to do this for the other migrated apps whose IDs changed, but at this point almost everything from eos-apps has been retired, which is why I just scoped this to this Firefox-specific script.